Core/Core/Inbox/Addressbook/View/AddressbookRoleView.swift

```diff
@@ -0,0 +1,168 @@
+            placement: .navigationBarDrawer(displayMode: .always)
+        )
+        .refreshable {
+            await viewModel.refresh()
+        }
+        .background(Color.backgroundLightest)
+        .navigationTitle(viewModel.title)
+        .navigationBarItems(leading: cancelButton)
+    }
+
+    private var loadingIndicator: some View {
+        ProgressView()
+            .progressViewStyle(.indeterminateCircle())
+            .frame(maxWidth: .infinity, maxHeight: .infinity)
+            .accentColor(Color(Brand.shared.primary))
+            .padding()
+    }
+
+    private var separator: some View {
+        Color.borderMedium
+            .frame(height: 0.5)
+    }
+
+    private var cancelButton: some View {
+        Button {
+            viewModel.cancelButtonDidTap.accept(controller)
+        } label: {
+            Text("Cancel", bundle: .core)
+                .font(.regular16)
+                .foregroundColor(.accentColor)
+        }
+    }
+
+    private var peopleView: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            separator
+            if viewModel.isAllRecipientButtonVisible { allRecipient }
+            ForEach(viewModel.recipients, id: \.self) { user in
+                personRowView(user)
+            }
+        }
+    }
+
+    private func personRowView(_ recipient: SearchRecipient) -> some View {
+        VStack(alignment: .leading, spacing: 0) {
+            Button(action: {
+                viewModel.recipientDidTap.send((recipient: [recipient], controller: controller))
+            }, label: {
+                HStack(alignment: .center, spacing: 16) {
+                    Avatar(name: recipient.displayName, url: recipient.avatarURL, size: 36, isAccessible: false)
+                    Text(recipient.displayName ?? recipient.fullName)
+                        .font(.regular16)
+                        .foregroundColor(.textDarkest)
+                        .lineLimit(1)
+                        .frame(maxWidth: .infinity, alignment: .leading)
+                }
+            })
+            .padding(16)
+            separator
+        }
+    }
+
+    private var rolesView: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            separator
+            allRecipient
+            ForEach(viewModel.roles, id: \.self) { role in
+                roleRowView(role)
+            }
+        }
+    }
+
+    private func roleRowView(_ role: String) -> some View {
+        VStack(alignment: .leading, spacing: 0) {
+            Button(action: {
+                viewModel.roleDidTap.send((roleName: role, recipient: viewModel.roleRecipients[role] ?? [], controller: controller))
+            }, label: {
+                HStack(alignment: .center, spacing: 16) {
+                    Avatar(name: role, url: nil, size: 36, isAccessible: false)
+                    VStack(alignment: .leading) {
```

Comment:
Let's add an explicit spacing. The one currently used seems to be too small compared to the one in figma and also could change in the future.